templates/js-start-scrape-single-page/.actor/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Specify the base Docker image. You can read more about
+# the available images at https://docs.apify.com/sdk/js/docs/guides/docker-images
+# You can also use any other image from Docker Hub.
+FROM apify/actor-node:16
```

Comment:
I wonder if maybe we should take this opportunity and bump all templates to Node 18? It's LTS and 16 is pretty old already.

Reply:
I was thinking the same, v16 is going EOL in 5 months.